### PR TITLE
WebTorrent Desktop v0.17.0

### PR DESCRIPTION
**Pull request by dcposch**
@feross @Flet @grunjol @mathiasvr

### CODE REVIEW

**Comment by Flet** [comment]
👍 with #964 merged :)

**Comment by dcposch** [comment]
Sounds good. Checklist for me
- [ ] Improve checkbox UI
  - [ ] The first time a user opens the app, it shouldn't say "Paused" five times
  - [ ] Clicking Play should not automatically check the checkbox
- [ ] Rebase on master to get those fixes plus the saveBounds fix
- [ ] Test
- [ ] Ship 🚢 🚢 

@feross

**Comment by dcposch** [comment]
### Design

#### No 'Paused' text on new torrents

This makes the app look cleaner the first time you run it after installing:

screen shot 2016-09-25 at 5 57 34 pm

See #966 

#### Smaller progress bar

Less visual weight.

screen shot 2016-09-25 at 5 49 30 pm

#### Expand and collapse handle

We had a user file an issue, saying they want to see the list of files in a torrent.

We want to make sure new users know they can click on a torrent to see details.

screen shot 2016-09-25 at 5 56 26 pm

**Comment by feross** [comment]
LGTM, ship it 🚢

File: static/main.css [hunk 1]
```diff
@@ -398,6 +398,24 @@ textarea,
...
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+.torrent:hover hr {
+  display: block;
+}
+
+>>>>>>> 9055623... Design: torrent list expand and collapse handle
```
Comment (feross) [inline static/main.css]: What is this doing here?

Comment (dcposch) [inline static/main.css]: Fixed

**Comment by grunjol** [comment]
The empty checkbox kind of confused me... without a label or a start button its not a clear indicator to what to do

**Comment by grunjol** [comment]
It even hasnt a tooltip togive youa clue

**Comment by demoneaux** [comment]
Maybe sometime like a "click to start" for first time users?

**Comment by feross** [comment]
Yeah, we can keep honing this in future PRs. I like the idea of showing "Click to start" next to the checkbox on hover

**Pull request merged**